caddyhttp/proxy/proxy_test.go

```diff
@@ -1470,3 +1472,57 @@ func TestChunkedWebSocketReverseProxy(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestQuic(t *testing.T) {
+	upstream := "quic.clemente.io:8086"
+	config := "proxy / quic://" + upstream
+	content := "Hello, client"
+
+	// make proxy
+	upstreams, err := NewStaticUpstreams(caddyfile.NewDispenser("Testfile", strings.NewReader(config)), "")
+	if err != nil {
+		t.Errorf("Expected no error. Got: %s", err.Error())
+	}
+	p := &Proxy{
+		Next:      httpserver.EmptyNext, // prevents panic in some cases when test fails
+		Upstreams: upstreams,
+	}
+
+	// start QUIC server
+	go func() {
+		dir, err := os.Getwd()
+		if err != nil {
+			t.Errorf("Expected no error. Got: %s", err.Error())
+			return
+		}
+		certDir := path.Join(dir, "../../vendor/github.com/lucas-clemente/quic-go/example/")
```

Comment:
Aha -- this won't work on Windows. You need to use `filepath.Join(...)` where each argument is one element of the path. Go will use the proper seperator for the OS.

Hrm, not really sure I like this hard-coded path. Is it possible to copy just the certificate directory into a local testdata folder?

Reply:
The windows API's take both forward and backslashes

Reply:
Oh. Then I was wrong. 😅 Maybe moving the certs I to a local folder is still a good idea though.